configs/JSC-JUWELS-Booster-rome-A100-Infiniband/auxiliary_software_files/compilers.yaml

```
@@ -0,0 +1,34 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+compilers:
+- compiler:
+      spec: nvhpc@=23.7
+      paths:
+        cc: /p/software/juwelsbooster/stages/2024/software/NVHPC/23.7-CUDA-12/Linux_aarch64/23.7/compilers/bin/nvc
+        cxx: /p/software/juwelsbooster/stages/2024/software/NVHPC/23.7-CUDA-12/Linux_aarch64/23.7/compilers/bin/nvc++
+        f77: /p/software/juwelsbooster/stages/2024/software/NVHPC/23.7-CUDA-12/Linux_aarch64/23.7/compilers/bin/nvfortran
+        fc: /p/software/juwelsbooster/stages/2024/software/NVHPC/23.7-CUDA-12/Linux_aarch64/23.7/compilers/bin/nvfortran
+      operating_system: rocky9
```

```suggestion
      operating_system: rocky8
```

JUWELS Booster is using Rocky 8, c.f. 

```
[jwlogin21 ~]$ cat /etc/os-release | grep PRETTY_NAME
PRETTY_NAME="Rocky Linux 8.9 (Green Obsidian)"
```